rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -214,11 +221,13 @@
             @QueryParam("running") @DefaultValue("true") boolean running,
             @QueryParam("finished") @DefaultValue("true") boolean finished,
             @QueryParam("childJobs") @DefaultValue("true") boolean childJobs,
+            @QueryParam("jobName") @DefaultValue("") String jobName,
+            @QueryParam("projectName") @DefaultValue("") String projectName,
```

Comment:
projectName should not have default value empty, as empty project can be a filter criteria